.github/workflows/erdpy-publish.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       run: |
-        echo " - [CHANGELOG](https://github.com/ElrondNetwork/elrond-sdk-erdpy/blob/main/erdpy/CHANGELOG.md)" >> notes.txt
+        echo " - [CHANGELOG](https://github.com/multiversx/mx-sdk-erdpy/blob/main/erdpy/CHANGELOG.md)" >> notes.txt
```

Comment:
Remove this link (since _changelog_ has been removed). Actually, remove usage of `notes.txt` all-together?

For example:
https://github.com/multiversx/mx-sdk-erdjs/pull/247/files#diff-196c0cb5a15344d79f17f60d56331cfc388e9f5cbe28f8350f38724ad06aa139

Reply:
Done!